### PR TITLE
Longer extension service worker lifetimes.

### PR DESCRIPTION
**Pull request by jpmedley**
Publication targeted for Friday, January 27.

### CODE REVIEW

**Review by IanStanion-google — APPROVED** [review]
I did a copy-pass and don't see anything wrong!

File: site/en/blog/longer-esw-lifetimes/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  Extension service workers can now stay alive as long as they're receiving events. This increases the reliability of extension services workers, but has a pitfall you should avoid. 
+subhead: >
+  Extension service workers can now stay alive as long as they're receiving events. This increases the reliability of extension services workers, but has a pitfall you should avoid.
+date: 2023-01-25
+authors:
+joemedley
+tags:
+  - extensions-news
+  - chrome110
+hero: sQ51XsLqKMgSQMCZjIN0B7hlBO02/iND6W98T4Ve3pWoj4Xq5.jpeg
```
Comment (jpmedley) [inline site/en/blog/longer-esw-lifetimes/index.md:14]:
```suggestion
hero: image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/iND6W98T4Ve3pWoj4Xq5.jpeg
```

File: site/en/blog/longer-esw-lifetimes/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Longer extension service worker lifetimes
+description: >
+  Extension service workers can now stay alive as long as they're receiving events. This increases the reliability of extension services workers, but has a pitfall you should avoid. 
+subhead: >
+  Extension service workers can now stay alive as long as they're receiving events. This increases the reliability of extension services workers, but has a pitfall you should avoid.
+date: 2023-01-25
+authors:
+joemedley
```
Comment (jpmedley) [inline site/en/blog/longer-esw-lifetimes/index.md:10]:
```suggestion
  - joemedley
```

File: site/en/blog/longer-esw-lifetimes/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - joemedley
+tags:
+  - extensions-news
+  - chrome110
+hero: image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/iND6W98T4Ve3pWoj4Xq5.jpeg
+alt: >
+  A picture of an hourglass.
+---
+
+Starting in Chrome 110, extension service workers stay alive as long as they're receiving events. This corrects a timing problem in the previous implementation of extension service workers. It was possible for timeouts to occur when new events were in the event queue and for the timeouts to truncate asynchronous work. This improvement removes the hard five-minute maximum lifetime for extension service workers.
```
Comment (jpmedley) [inline site/en/blog/longer-esw-lifetimes/index.md:19]:
```suggestion
Starting in Chrome 110 (in beta as of February 7, 2023), extension service workers stay alive as long as they're receiving events. This corrects a timing problem in the previous implementation of extension service workers. It was possible for timeouts to occur when new events were in the event queue and for the timeouts to truncate asynchronous work. This improvement removes the hard five-minute maximum lifetime for extension service workers. 
```

**Pull request merged**